core/base/inc/TString.h

```diff
@@ -309,15 +309,8 @@ friend std::strong_ordering operator<=>(const TString &s1, const TString &s2) {
 
    // Type conversion
    operator const char*() const { return GetPointer(); }
-#if (__cplusplus >= 201700L) && !defined(_MSC_VER) && (!defined(__clang_major__) || __clang_major__ > 5)
```

Comment:
How certain are we that Windows doesn't need this anymore? I guess the CI will tell...

Reply:
Yes :)

Reply:
Okay, it's still needed. I don't like that it's there, but that would be another story